src/js/auth.js

```diff
@@ -49,12 +73,12 @@ document.addEventListener("DOMContentLoaded", function () {
 
         const API_LOGIN_URL = "https://api.noroff.dev/api/v1/social/auth/login";
```

Comment:
The actual base url (https://api.noroff.dev/api/v1/social/) should be imported from a constants file.